cassandra/delete_data.py

```diff
@@ -25,43 +30,71 @@ def __init__(self, refdes, uframe_ip, cassandra_ip_list):
         self.refdes = refdes
         self.subsite, self.node, self.sensor = self.parse_refdes(refdes)
         # For now use version=3 against the current cassandra.
-        cluster = cassandra.cluster.Cluster(cassandra_ip_list, control_connection_timeout=60, protocol_version=3)
+        cluster = cassandra.cluster.Cluster(
+                cassandra_ip_list, control_connection_timeout=60,
+                protocol_version=3)
         self.session = cluster.connect('ooi')
-        stream_url = STREAM_METADATA_SERVICE_URL_TEMPLATE.format(uframe_ip)
-        partition_url = PARTITION_METADATA_SERVICE_URL_TEMPLATE.format(uframe_ip)
-        self.metadata_service_api = MetadataServiceAPI(stream_url, partition_url)
+        stream_url = STREAM_METADATA_SERVICE_URL_TEMPLATE.format(
+                uframe_ip)
+        partition_url = PARTITION_METADATA_SERVICE_URL_TEMPLATE.format(
+                uframe_ip)
+        self.metadata_service_api = MetadataServiceAPI(
+                stream_url, partition_url)
+        doi_url = DOI_SERVICE_URL_TEMPLATE.format(uframe_ip)
+        self.doi_service_api = DOIServiceAPI(doi_url)
 
     @staticmethod
     def parse_refdes(refdes):
         return refdes.split('-', 2)
 
     def get_stream_info(self):
         result = {}
-        partition_metadata_record_list = self.metadata_service_api.get_partition_metadata_records(
-            self.subsite, self.node, self.sensor
-        )
+        partition_metadata_record_list = (
```

Comment:
My PEP8 checking tool was trying to enforce a line length of 80 which forced some weird formatting like extra parentheses and triple quotes to have proper line continuation.